lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -6383,6 +6384,20 @@ LLVMToSPIRVBase::transBuiltinToInstWithoutDecoration(Op OC, CallInst *CI,
     return BM->addCompositeConstructInst(transType(CI->getType()), Operands,
                                          BB);
   }
+  case OpIAddCarry: {
+    Function *F = CI->getCalledFunction();
+    StructType *St = cast<StructType>(F->getParamStructRetType(0));
+    return BM->addBinaryInst(OpIAddCarry, transType(St),
```

Comment:
Like this it will be a dangling OpIAddCarry, we need to store the result to the original sret pointer